packages/DropdownMenu/src/DropdownMenu.js

```diff
@@ -133,22 +133,24 @@ function DropdownMenu(props) {
     }
 
     return (
-      <div css={contentStyles} ref={dropdownListRef}>
-        {extractedChildren.children.map((child, index) => {
-          const childKey = { key: `DropdownMenuItem${index}` };
-          if (child && child.type && child.type.displayName === "DropdownMenu.Item") {
-            if (child.props.renderConfirmation) {
+      <Popover.Card>
```

Comment:
Only wrapping the content with the popover card when it is the dropdown list of items, and not when it is just showing the confirmation panel. This avoids the rendering of the card (with no contents) when we are closing the confirmation panel.